FirebaseInstallations/REQUIRED_FIREBASE_OPTIONS_IOS.md

```diff
@@ -0,0 +1,40 @@
+# Firebase services require valid Firebase options
+## Firebase Options
+[Firebase options](https://firebase.google.com/docs/reference/ios/firebasecore/api/reference/Classes/FIROptions) are a set of parameters required by services in order to successfully communicate with Firebase server APIs and in order to associate client data with your Firebase project and Firebase application.
+
+Firebase services rely on valid Firebase options being available from the Firebase core SDK [`FIRApp`](https://firebase.google.com/docs/reference/ios/firebasecore/api/reference/Classes/FIRApp), created during Firebase initialization.
+
+Different Firebase services require different Firebase options to function properly, but all Firebase services require the following Firebase options:
+* [**API key**](https://cloud.google.com/docs/authentication/api-keys) - Note: this is **not** an FCM server key, see below. \
+  Example value: `AIzaSyDOCAbC123dEf456GhI789jKl012-MnO`
+* [**Project ID**](https://firebase.google.com/docs/projects/learn-more#project-id) \
+  Example value: `ios-myapp-123`
+* **Application ID**, a.k.a. `GOOGLE_APP_ID`, `mobilesdk_app_id`, "AppId" - Note: this is **not** an Android package name! \
+  Example value: `1:1234567890:ios:321abc456def7890`
+
+## What do I need to know?
+To improve security Firebase SDK updates [on January 14](https://firebase.google.com/support/release-notes/ios#version_6150_-_january_14_2020) and afterwards replaced the Firebase Instance ID service with a dependency on the [Firebase Installations API](https://console.cloud.google.com/apis/library/firebaseinstallations.googleapis.com).
+
+Firebase Installations enforces the existence and validity of mandatory Firebase options API key, Project ID, and Application ID in order to associate client data with your Firebase project.
+
+## Firebase Cloud Messaging (FCM) with Firebase Instance ID (IID)
+If you are reading this message, most likely your application is initializing Firebase without the required set of Firebase options.
```

Comment:
Should there be an acknowledgement that the apps were silently malfunctioning even before the transition to FirebaseInstallations?